src/helpers/helpers.ts

```diff
@@ -0,0 +1,16 @@
+import { FilterStatus } from '../types/Filterstatus';
+import { Todo } from '../types/Todo';
+
+export const filterTodosByCompleted = (
+  todos: Todo[], filterStatus: FilterStatus,
+) => todos
+  .filter(todo => {
+    switch (filterStatus) {
+      case FilterStatus.Active:
+        return !todo.completed;
+      case FilterStatus.Completed:
+        return todo.completed;
+      default:
```

Comment:
But better to check filter type before filtering

```suggestion
) => 
switch (filterStatus) {
  case FilterStatus.Active:
    return todos.filter(todo => (
      !todo.completed;
    )
```